lib/yoti/doc_scan/session/create/requested_document_authenticity_check.rb

```diff
@@ -30,20 +30,62 @@ def self.builder
         # The configuration applied when creating a {RequestedDocumentAuthenticityCheck}
         #
         class RequestedDocumentAuthenticityCheckConfig
+          #
+          # @param [String] manual_check
+          #
+          def initialize(manual_check)
+            Validation.assert_is_a(String, manual_check, 'manual_check', true)
+            @manual_check = manual_check
+          end
+
           def as_json(*_args)
-            {}
+            {
+              manual_check: @manual_check
+            }.compact
           end
         end
 
         #
         # Builder to assist the creation of {RequestedDocumentAuthenticityCheck}
         #
         class RequestedDocumentAuthenticityCheckBuilder
+          #
+          # Requires that a manual follow-up check is always performed
+          #
+          # @return [self]
+          #
+          def with_manual_check_always
+            @manual_check = Constants::ALWAYS
+            self
+          end
+
+          #
+          # Requires that a manual follow-up check is performed only on failed Checks,
```

Comment:
Should we make `check` lower case? I think that's the approach we've taken in the docs. It's lower case in method above, and upper in method below
```suggestion
          # Requires that a manual follow-up check is performed only on failed checks,
```

Reply:
Fixed in 7716578